assets/maps/maps.ldtk

```diff
@@ -354,19 +354,19 @@
 					"intGridCsv": [
 						0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
 						0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
+						0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,0,1,
```

Comment:
I'm going to find out how to hide these files form the PR diffs because it's not useful. :thinking:

Reply:
f50a9aa37212f229ba356a88ea3d9238e2200c87 :muscle: